src/app/store.ts

```diff
@@ -1,6 +1,9 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
The function `combineSlices` is not a standard function from `@reduxjs/toolkit`. It should be `combineReducers` instead. Please verify if `combineSlices` is a custom utility or replace it with `combineReducers`.